orca-0/Dockerfile

```diff
@@ -25,6 +25,7 @@ less \
 libxml2 \
 matplotlib \
 miller \
+mysql \
```

Comment:
I don't think we need `mysql` in the image if nothing is using it.

Reply:
Done!